test/unit/jwt_token_test.rb

```diff
@@ -31,7 +31,8 @@ class JwtTokenTest < ActiveSupport::TestCase
 
     expected = {'user_id' => 123,
                 'iat' => 1_514_804_400,
-                'jti' => '3e8286940eb162ec735f8a5aac5926037fdc7f05e521a74231a65f2557a8af94'}
+                'jti' => '3e8286940eb162ec735f8a5aac5926037fdc7f05e521a74231a65f2557a8af94',
+                'scope' => []}
 
     assert_equal expected, jwt_token.decode
   end
```

Comment:
Can you add a test where you have the same user id, iat but different scopes (A and B) and check if the generated hash (jti) is different? I want to ensure that scope is subject of the security hash.